### PR TITLE
[CLOUD-1966] (Android) Add plugin API to check battery optimizations status

### PR DESCRIPTION
**Pull request by koichirosobue**
https://ischemaview.atlassian.net/browse/CLOUD-1966

- Forked repository from https://github.com/katzer/cordova-plugin-background-mode
- Added Jenkins file for Nexus integration
- Added `isBatteryOptimizationsDisabled()` method which returns the status of battery optimizations option for the app in Android.

### CODE REVIEW

File: src/android/BackgroundModeExt.java [hunk 1]
```diff
@@ -185,6 +188,22 @@ public void run() {
...
     }
 
+    /**
+     * Checks if battery optimizations is disabled for the app.
+     */
+    @SuppressLint("BatteryLife")
+    private boolean isBatteryOptimizationsDisabled(CallbackContext callback)
+    {
+        Activity activity = cordova.getActivity();
+        Intent intent     = new Intent();
```
Comment (nirajrajbhandari) [inline src/android/BackgroundModeExt.java]: is this required?

File: package.json [hunk 2]
```diff
@@ -37,5 +37,8 @@
   "bugs": {
     "url": "https://github.com/katzer/cordova-plugin-background-mode/issues"
   },
-  "homepage": "https://github.com/katzer/cordova-plugin-background-mode#readme"
+  "homepage": "https://github.com/katzer/cordova-plugin-background-mode#readme",
```
Comment (nirajrajbhandari) [inline package.json]: should we change the homepage URL?

Comment (koichirosobue) [reply package.json]: I'm going to make a PR of `isBatteryOptimizationsDisabled()` to the official repo. This repo is temporary until it's merged.

**Comment by koichirosobue** [comment]
@suryadroid I'm going to make a PR for `isBatteryOptimizationsDisabled()` method to the official repo later. This repo is temporary until the official repo merges my change.

**Pull request merged**